tasks/src/const.ts

```diff
@@ -6,35 +6,45 @@ import type { PipelineType } from "../../js/src/lib/interfaces/Types";
  */
 export const TASKS_MODEL_LIBRARIES: Record<PipelineType, Array<keyof typeof ModelLibrary>> = {
 	/// nlp
-	"text-classification":            ["adapter-transformers", "spacy", "transformers"],
-	"token-classification":           ["adapter-transformers", "flair", "spacy", "stanza", "transformers"],
-	"table-question-answering":       ["transformers"],
+	"fill-mask":                      ["transformers"],
 	"question-answering":             ["adapter-transformers", "allennlp", "transformers"],
-	"zero-shot-classification":       ["transformers"],
-	"translation":                    ["transformers"],
 	"summarization":                  ["transformers"],
-	"conversational":                 ["transformers"],
-	"feature-extraction":             ["sentence-transformers", "transformers"],
+	"table-question-answering":       ["transformers"],
+	"text-classification":            ["adapter-transformers", "spacy", "transformers"],
 	"text-generation":                ["transformers"],
 	"text2text-generation":           ["transformers"],
-	"fill-mask":                      ["transformers"],
+	"token-classification":           ["adapter-transformers", "flair", "spacy", "stanza", "transformers"],
+	"translation":                    ["transformers"],
+	"zero-shot-classification":       ["transformers"],
 	"sentence-similarity":            ["sentence-transformers", "spacy"],
-	/// audio
-	"text-to-speech":                 ["espnet", "tensorflowtts"],
+	"conversational":                 ["transformers"],
+	"tabular-classification":         ["sklearn"],
+	"tabular-to-text":                ["transformers"],
+	"table-to-text":                  ["transformers"],
+	"multiple-choice":                ["transformers"],
+	"text-retrieval":                 [],
+	/// multimodal
+	"feature-extraction":             ["sentence-transformers", "transformers"],
 	"automatic-speech-recognition":   ["espnet", "speechbrain", "transformers"],
+	"text-to-speech":                 ["espnet", "tensorflowtts"],
+	"text-to-image":                  [],
+	"image-to-text":                  [],
+	"visual-question-answering":      [],
+	"zero-shot-image-classification":  [],
+	/// audio
```

Comment:
i would order those by their order in `PIPELINE_DATA` rather than in `PIPELINE_TAGS_DISPLAY_ORDER`,  because the former is way less likely to change in the future

Reply:
(or just alphabetically if it's simpler)

Reply:
Let's go with alphabetically so it keeps us sane :)